packages/tools/examples/nonVolumeNavigationTools/index.ts

```diff
@@ -0,0 +1,275 @@
+    });
+  },
+});
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  const toolGroupId = 'NAVIGATION_TOOL_GROUP_ID';
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(PanTool);
+  cornerstoneTools.addTool(RotateTool);
+  cornerstoneTools.addTool(ZoomTool);
+
+  // Define a tool group, which defines how mouse events map to tool commands for
+  // Any viewport using the group
+  const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+  // Add tools to the tool group
+  toolGroup.addTool(PanTool.toolName);
+  toolGroup.addTool(RotateTool.toolName);
+  toolGroup.addTool(ZoomTool.toolName);
+
+  // Set the initial state of the tools, here all tools are active and bound to
+  // Different mouse inputs
+  toolGroup.setToolActive(PanTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Auxiliary, // Middle + Drag
+      },
+    ],
+  });
+
+  toolGroup.setToolActive(RotateTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Left Click + Drag
+      },
+    ],
+  });
+
+  toolGroup.setToolActive(ZoomTool.toolName, {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Secondary, // Right Click + Drag
+      },
+    ],
+  });
+
+  // Instantiate a rendering engine
+  const renderingEngine = new RenderingEngine(renderingEngineId);
+
+  // Create the viewports
+  const viewportInputArray = [
+    {
+      viewportId: viewportId1,
+      type: ViewportType.ORTHOGRAPHIC,
+      element: element1,
+      defaultOptions: {
+        orientation: ORIENTATION.SAGITTAL,
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+    {
+      viewportId: viewportId2,
+      type: ViewportType.PERSPECTIVE,
+      element: element2,
+      defaultOptions: {
+        orientation: ORIENTATION.SAGITTAL,
+        background: <Types.Point3>[0, 0.2, 0.2],
+      },
+    },
+  ];
+
+  renderingEngine.setViewports(viewportInputArray);
+
+  // Set the tool group on the viewport
+  toolGroup.addViewport(viewportId1, renderingEngineId);
+  toolGroup.addViewport(viewportId2, renderingEngineId);
+
+  viewportIds.forEach((viewportId) => {
+    // Get the stack viewport that was created
+    const viewport = <Types.IVolumeViewport>(
+      renderingEngine.getViewport(viewportId)
+    );
+
+    const sphereSource = vtkSphereSource.newInstance({
+      center: [0, 0, 0],
+      radius: 100,
+      phiResolution: 10,
+      thetaResolution: 10,
+    });
+    const actor = vtkActor.newInstance();
+    const mapper = vtkMapper.newInstance();
+
+    actor.getProperty().setEdgeVisibility(true);
+
+    mapper.setInputConnection(sphereSource.getOutputPort());
+    actor.setMapper(mapper);
+
+    const nonVolumeActors = [];
+    nonVolumeActors.push({ uid: 'spherePolyData', actor });
+
+    viewport.setActors(nonVolumeActors);
+
+    // Render the image
+    viewport.render();
+  });
+}
+
+run();
```

Comment:
Same comments as the example in the core demo, we don't need any other view for sphere, and not much toolbar. In my opinion, the purpose of these example is how easy it is to render + manipulate the polyData actor in some couple of lines of code